Prace_domowe/Praca_domowa2/Jan_Gaska/WUM - PD2.ipynb

```diff
@@ -0,0 +1,2443 @@
+{
```

Comment:
<p>Przed dokonaniem analizy dot. unikalnych wartości należałoby dokonać minimalnej unifikacji wartości typu string poprzez np. zamianę wszystkich wartości na małe litery - może być tak, że raz stolica w danych została zapisana jako Warszawa a raz jako warszawa a de facto jest to to samo miasto więc nie ma potrzeby aby miały inny encoding.</p><p> </p><p>Z uwagi na to, że kwestia ta była poruszana na zajęciach, będę musiała niestety odjąć -0.5 pkt.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>